### PR TITLE
Document ActionDispatch::TestResponse#parsed_body [ci-skip]

### PR DESCRIPTION
**Pull request by shouichi**
I found this little gem. This deserves documentation.

### CODE REVIEW

**Review by jonathanhefner — COMMENTED** [review]
Nice! :+1:

File: actionpack/lib/action_dispatch/testing/test_response.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@ def self.from_response(response)
       new response.status, response.headers, response.body
     end
 
+    # Returns a parsed body depending on the response mime-type. When
+    # corresponding parser for mime-type is not found, it returns the raw body.
```
[the remark this answers is not in the document]
Comment (shouichi) [reply actionpack/lib/action_dispatch/testing/test_response.rb]: Adding examples would be great! I'll add them.

**Comment by shouichi** [comment]
@jonathanhefner Thanks for the feedback! I committed the suggestion and added examples.

File: actionpack/lib/action_dispatch/testing/test_response.rb [hunk 2]
```diff
@@ -14,6 +14,24 @@ def self.from_response(response)
...
+    #
+    #   get "/posts.json"
+    #   response.content_type      # => "application/json; charset=utf-8"
+    #   response.parsed_body.class # => Array
+    #   response.parsed_body       # => [{"id"=>42, "title"=>"Title"},...
+    #
+    #   get "/posts/42.json"
+    #   response.content_type      # => "application/json; charset=utf-8"
+    #   response.parsed_body.class # => Hash
+    #   response.parsed_body       # => {"id"=>42, "title"=>"Title"}
```
Comment (jonathanhefner) [inline actionpack/lib/action_dispatch/testing/test_response.rb:34]: Excellent! :heart:

**Comment by jonathanhefner** [comment]
@shouichi Looks great!  Would you mind also [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)? :pray:  Then I will give this a merge!

**Comment by shouichi** [comment]
@jonathanhefner squashed!

**Comment by jonathanhefner** [comment]
Thank you, @shouichi! :smile:

**Pull request merged**